target/linux/rtl838x/files-5.4/drivers/gpio/Kconfig

```diff
@@ -0,0 +1,1500 @@
+# SPDX-License-Identifier: GPL-2.0-only
```

Comment:
Please do not overwrite this file, but add a patch with your changes.

Reply:
Will do.

Reply:
Done.